channels.yaml

```diff
@@ -427,11 +440,12 @@ releases:
   - version: v1.26.6+k3s1
     minChannelServerVersion: v2.7.5-alpha1
     maxChannelServerVersion: v2.7.99
-    serverArgs: &serverArgs-v6
-      <<: *serverArgs-v4
-      vpn-auth:
-        type: string
-      vpn-auth-file:
-        type: string
+    serverArgs: *serverArgs-v3
+    agentArgs: *agentArgs-v3
+    featureVersions: *featureVersions-v1
+  - version: v1.26.7+k3s1
+    minChannelServerVersion: v2.7.5-alpha1
+    maxChannelServerVersion: v2.7.99
+    serverArgs: *serverArgs-v6
```

Comment:
This should probably pull from `serverArgs-v4` and then include the `helm-job-image` to eventually have its own `&serverArgs-v6`